### DISCUSSION TITLE
Overall projects dashboard?

### DISCUSSION THREAD

**Discussion by justingodden** [langfuse/langfuse#2599] [Support] [answered]
Hi, is it currently possible, or is there plans to add to be able to have an overall dashboard that shows metrics for all projects?

It would be handy to see an aggregated version (i.e. total cost, tokens etc. across all projects), and also comparison charts, similar to the current dashboard, but comparing cost, latency, tokens, number of traces by project.

Thanks

**Comment by marcklingen** [comment, marked as answer]
Currently this is not possible, thanks for sharing this idea! We are tracking this idea here: https://github.com/orgs/langfuse/discussions/1258
